docs/templates/orpc-service.md

```
@@ -39,3 +39,7 @@ export default {
   },
 };
 ```
+
+::: tip Need something else?
+If this template doesn't cover what you need, DM me on X (https://x.com/omardulaimidev) and we can scope it together.
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)